### ISSUE TITLE
bgp_session_info . oracle_bgp_asn is returning empty string in oci_core_ipsec_connection_tunnel_management resource

### ISSUE THREAD

**Issue by imrannayer** [oracle/terraform-provider-oci#1517]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version and Provider Version

Terraform v1.0.5
### Affected Resource(s)

oci_core_ipsec_connection_tunnel_management resource

### Terraform Configuration Files

```hcl
resource "oci_core_ipsec_connection_tunnel_management" "ipsec_connection_0_tunnel_0" {
  ipsec_id  = oci_core_ipsec.ip_sec_connection_0.id
  tunnel_id = data.oci_core_ipsec_connection_tunnels.ip_sec_connection_0_tunnels.ip_sec_connection_tunnels[0].id

  #Optional
  bgp_session_info {
    customer_bgp_asn      = var.customer_router_asn
    customer_interface_ip = var.customer_router_ip
    oracle_interface_ip   = var.bgp_oci_interface_ip_0
  }

  display_name  = "${var.prefix}-ipsec0-tunnel0"
  routing       = "BGP"
  shared_secret = random_string.secret.result
  ike_version   = "V2"
}

output ipsec_connection_0_tunnel_0 {
value = oci_core_ipsec_connection_tunnel_management.ipsec_connection_0_tunnel_0.bgp_session_info[0].oracle_bgp_asn
}
```

### Expected Behavior

Output should should show ASN number.
### Actual Behavior

It shows empty string
### Steps to Reproduce

1. `terraform apply`

### References
https://registry.terraform.io/providers/hashicorp/oci/latest/docs/resources/core_ipsec_connection_tunnel_management#attributes-reference

**Comment by iejones** [comment]
A bug fix to properly set oracle_bgp_asn will be released on November 15th, 2023

**Comment by iejones** [comment]
This fix was released with version [v5.21.0](https://github.com/oracle/terraform-provider-oci/releases/tag/v5.21.0)
